.github/workflows/release-please.yml

```diff
@@ -0,0 +1,48 @@
+# Automate releases of new app versions
+name: release-please
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
By the way, why is `master` not listed here? Is it because we know that the branch never existed?

Reply:
Yes the template workflow has the special `$default-branch` (see [GitHub docs](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization#creating-a-starter-workflow)) which resolves to the repo default branch (main in this case) I added `master` in the repos that still use it, but i think we should move towards renaming those 😄